source/documentation/appendix-docs/botor.md

```diff
@@ -570,6 +570,7 @@ list_files_in_buckets <- function(bucket_filter = NULL, prefix = NULL,
   }
   file_list <- dplyr::bind_rows(purrr::map(bucket_filter, 
                                            list_files_in_bucket))
+  if(is.numeric(max)) file_list <- head(file_list, max)
```

Comment:
For full compatibility this needs to be in the `list_files_in_bucket` function - see https://github.com/moj-analytical-services/s3tools/blob/934c9f9420acb168107979267660b2dbd31b6cd7/R/aws_helper_functions.R#L36 - but I'm not sure if people use it. Maybe if they have a large number of small data files.